src/NovelRT.Interop/Audio/NrtAudioService.cpp

```diff
@@ -116,7 +189,16 @@ NrtResult Nrt_AudioService_setMusicVolume(NrtAudioService service, float value)
     }
 
     auto serv = reinterpret_cast<Audio::AudioService*>(service);
-    serv->setMusicVolume(value);
+
+    try
+    {
+      serv->setMusicVolume(value);
+    }
+    catch( const Exceptions::NotInitialisedException)
+    { // todo: handle error message
```

Comment:
This looks like a custom exception. Do we have an internal function for the error message?